server/asset/dcr/dcr.go

```diff
@@ -613,3 +689,9 @@ func toCoinID(txHash *chainhash.Hash, vout uint32) []byte {
 func toAtoms(v float64) uint64 {
 	return uint64(math.Round(v * 1e8))
 }
+
+// isTxNotFoundErr will return true if the error indicates that the requested
+// transaction is not known.
+func isTxNotFoundErr(err error) bool {
+	return strings.HasPrefix(err.Error(), "-5:")
```

Comment:
I hate it, but it might be the best option for right now.

Reply:
Yeah, OK for now.  Although, I suspect a `errors.As` with a `dcrjson/v3.RPCError` would be fine since that matches rpcclient's version of dcrjson.  As far as the node, I think that's were we would have RPC version checks.  Let's put a TODO on this line and revisit.